core/util/pastebin.py

```diff
@@ -0,0 +1,86 @@
+"""
+OWASP Maryam!
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+class main:
+	def __init__(self, q, limit=1, count=10):
+		""" pastebin link
+
+		q       : The query domain
+		limit   : Number of pages
+		"""
+		self.framework = main.framework
+		self.agent = 'Mozilla/5.0 (Linux; Android 5.1; AFTS Build/LMY47O) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/41.99900.2250.0242 Safari/537.36'
+		self.q = q
+		self.query = f'site:pastebin.com "{self.q}"'
+		self._pages = ''
+		self.bing = 'bing.com'
+		self.ask = 'www.ask.com'
+		self._domains = []
+		self.extract_url = 'https://pastebin.com/raw'
+		self.limit = limit
+		self.count = count
+		self._links = []
+
+	def run_crawl(self):
```

Comment:
Why didn't use the self.framework.ask(), self.framework.bing()?

Reply:
It did not work, don't know why. I did that at first. Plus, in this way, we can add urls to pastebin only. Right?
I can change too though